src/components/events/EventsSection.jsx

```diff
@@ -22,16 +23,15 @@ class EventsSection extends React.Component {
     db.collection('events').onSnapshot((querySnapshot) => {
       const events = [];
       querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, date } = doc.data();
-        let eventDate;
-        if (date) {
-          eventDate = new Date(date.seconds * 1000).toLocaleDateString();
-        }
+        const { eventTitle, numJudges, eventDate } = doc.data();
+        const date = typeof eventDate === 'object'
```

Comment:
`typeof evenDate === 'object'` seems dangerous: https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript

Depending on what exactly the check for obj is here. We can use lodash.isObject: https://stackoverflow.com/questions/34111902/why-do-lodashs-isobject-isplainobject-behave-differently-than-typeof-x